server/testing/src/main/java/org/apache/james/utils/IMAPMessageReader.java

```diff
@@ -94,6 +94,13 @@ public IMAPMessageReader select(String mailbox) throws IOException {
         return this;
     }
 
+    public IMAPMessageReader create(String mailbox) throws IOException {
```

Comment:
in MessageReader? maybe it's time to rename that class?

Reply:
TestIMAPClient?